docs/running-notebooks/listing-active-sessions/index.md

```diff
@@ -0,0 +1,15 @@
+# Listing active sessions
+
+You can list the active Jupyter notebooks sessions from the REANA
+command-line client [reana-client](https://pypi.org/project/reana-client/)
+in the same way you list workflows and additionally passing the
+`-s/--sessions` flag:
+
+```console
+$ reana-client list --sessions
+NAME         RUN_NUMBER   CREATED               SESSION_TYPE   SESSION_URI                                                                           SESSION_STATUS
+jupyter      1            2021-12-01T14:33:26   jupyter        https://reana.cern.ch/f8be55e4-5d18-43f9-b977-f773fdcab163?token=<your-reana-token>   created
```

Comment:
Should we use `example.org`?  Also hoping UUID is some local one...

Reply:
UUID is copied from the section `opening-sessions/index.md`, where it's used together with `reana.cern.ch`.